easycv/core/evaluation/ocr_eval.py

```diff
@@ -0,0 +1,282 @@
+# Modified from https://github.com/PaddlePaddle/PaddleOCR/tree/release/2.6/ppocr/metrics
+import string
+from collections import namedtuple
+
+import numpy as np
+from rapidfuzz.distance import Levenshtein
+from shapely.geometry import Polygon
+
+from .base_evaluator import Evaluator
+from .builder import EVALUATORS
+from .metric_registry import METRICS
+
+
+@EVALUATORS.register_module()
+class OCRDetEvaluator(Evaluator):
+
+    def __init__(self, dataset_name=None, metric_names=['hmean']):
+        self.iou_constraint = 0.5
+        self.area_precision_constraint = 0.5
+        super().__init__(dataset_name, metric_names)
+
+    def _evaluate_impl(self, predictions, labels, **kwargs):
+        pass
+
+    def evaluate_image(self, gt, pred):
```

Comment:
Why not put the implementation in evaluate_image into _evaluate_impl？

Reply:
done